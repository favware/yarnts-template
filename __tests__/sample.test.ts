import defaultSample, { sampleFunc } from '../src';

describe('Named Import Sample', () => {
  test('should work with named import', () => {
    expect(sampleFunc('sample-named')).toBe('sample-named');
  });

  test('should match snapshot', () => {
    expect(sampleFunc).toMatchSnapshot();
  });
});

describe('Default Import Sample', () => {
  test('should work with default import', () => {
    expect(defaultSample('sample-default')).toBe('sample-default');
  });

  test('should match snapshot', () => {
    expect(defaultSample).toMatchSnapshot();
  });
});
