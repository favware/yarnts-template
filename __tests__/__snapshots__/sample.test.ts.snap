// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Import Sample should match snapshot 1`] = `[Function]`;

exports[`Named Import Sample should match snapshot 1`] = `[Function]`;
